components/sup/src/manager/mod.rs

```diff
@@ -240,15 +246,15 @@ impl Manager {
         };
 
         let services = Arc::new(RwLock::new(Vec::new()));
-        let server = butterfly::Server::new(&cfg.gossip_listen,
-                                            &cfg.gossip_listen,
+        let server = butterfly::Server::new(sys.gossip_listen(),
+                                            sys.gossip_listen(),
                                             member,
                                             Trace::default(),
                                             ring_key,
                                             None,
                                             Some(&fs_cfg.data_path),
                                             Box::new(SuitabilityLookup(services.clone())))?;
-        outputln!("Butterfly Member ID {}", server.member_id());
+        outputln!("Supervisor Member-ID {}", sys.member_id);
```

Comment:
That's a nice tweak and more clearly communicate with the extra internal jargon.